blog/static/blog/css/admin.css

```
@@ -0,0 +1,607 @@
+    opacity: 1;
+    transform: translateY(-1px);
+}
+
+/* ===== 링크/본문 ===== */
+.post-title-link {
+    position: relative;
+    color: #111111;
+    text-decoration: none;
+    font-weight: 500;
+    display: inline-block;
+    word-break: break-word;
```

Stylelint (17.4.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)